opensrp-reveal/src/main/java/org/smartregister/reveal/view/DrawerMenuView.java

```diff
@@ -55,6 +55,7 @@
     private TextView facilityTextView;
     private TextView operatorTextView;
     private TextView p2pSyncTextView;
+    private TextView offlineMapTextView;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'offlineMapTextView' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)